[Space Ruin] - The outlet

Donglesplonge opened the pull request:
## About The Pull Request

This PR adds in a new space ruin, known as the outlet. the outlet is a clothing store in disrepair over its time drifting through space, why was it abandoned in the first place? explore and find out! i've spent my fair share of time meandering around space looking at ruins for fun, so i figured with mapping march going on i'd throw out my own ruin for fun. 

<details> 
<summary> SDMM screenshots of the map </summary>

![map overview with areas](https://user-images.githubusercontent.com/120208006/228141828-4a7311c9-8ce7-4107-9514-3e4b3f2457db.PNG)

![map overview without areas](https://user-images.githubusercontent.com/120208006/228141870-7df3b532-a83f-491a-b7ea-fbc483e6ee90.PNG)


</details>

<details>
<summary> in game screenshots of parts of the map </summary>

![storefront](https://user-images.githubusercontent.com/120208006/228142049-f6a6f660-8dcb-4599-9d38-45b2a22993b3.PNG)

![research rooms](https://user-images.githubusercontent.com/120208006/228142087-e728ce11-4dbd-4c71-9999-04abb3e4675f.PNG)

![cultinfluence](https://user-images.githubusercontent.com/120208006/228142120-9fe2475a-6317-4560-b794-1252347743be.PNG)


</details>

Big ups to chesh, vire, and the people of the coding channel for helping me out with questions i had about github, i appreciate you all greatly for it.

Ckey to receive rewards: Donglesplonge

## Why It's Good For The Game

space is a fun place to be sometimes, if the station goes down the toilet and you just want peace and to be alone for a part of the round you don an EVA suit and jump out an airlock and drift, adding more substance to it would help make people feel happy about being in it, and i hope to help accomplish that with this ruin addition. It has sustainable atmospherics and a powernet to help it stay up in case you need a breather from some of the more dangerous places.

## Changelog

:cl:
add: central commands meteorological divsion has picked up a new asteroid drifting in NT space, what secrets could it hold?
/:cl:

Donglesplonge committed:
the outlet

adds the core files for the ruin, including the fluff section, the ID trim datum, and the map itself

Donglesplonge committed:
adds in more core files

adds in the map_template file, also adds in the ruin areas.

Donglesplonge committed:
map adjustments

adjusts some issues with the maps previous iteration, including rerouting piping and wiring to be more efficient, removes the double fabric storage door, removing 2 reduntant APCs, adding a wire under the PACMAN so it functions correctly, and adding in some more damage and debris to the employees clothing rack area and power room to make it more lived in. also adjusts the doors to no longer rely on varedits and instead properly use access_helpers

Donglesplonge committed:
Merge remote-tracking branch 'upstream/master' into the-clothing-outlet

Donglesplonge committed:
Merge branch 'master' into the-clothing-outlet

Donglesplonge committed:
blacklist update because the file disappeared

the blacklist file disappeared while i was updating something and i somehow missed it when doing double checks, whoopsy

Donglesplonge committed:
finale room crate edit

hopefully resolves the compile map error via varedited crates by just using regular crates instead

Donglesplonge committed:
door tweaks to help compile maps

went and re-checked every single door for access errors and also moved the toilets around because i thought they looked weird.

Donglesplonge commented:
i'm marking this under draft instead of review so i can continue working on it, as apparently i couldn't catch all the errors using VSC, sorry for the inconvenience.

Donglesplonge committed:
Door naming update

updates door names to match what the linter gods request of me, also replaces the maintenance doors names to Electrical, to match what the room is used for.

Donglesplonge commented:
i overreacted a bit making it a draft thinking i had a large amount of errors but it turns out it wasn't

Donglesplonge committed:
Merge branch 'master' into the-clothing-outlet

Donglesplonge committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into the-clothing-outlet

Donglesplonge committed:
lore page rewriting

redoes some of the lore writing to make it come off a bit stronger than i had initially wrote it to be, also re-adjusts the name of chemicals_test fluffpaper to tanners_steps to better represent what the document is actually about.

Donglesplonge committed:
map paper update and lore paper fix

adds in the tanners_steps paper in place of chemical_tests, and also fixes extra space after the asterisk at the end of the paper, which should let it work now

Donglesplonge committed:
Merge branch 'master' into the-clothing-outlet

Donglesplonge committed:
Merge branch 'master' into the-clothing-outlet

Donglesplonge committed:
Merge remote-tracking branch 'upstream/master' into the-clothing-outlet

Donglesplonge committed:
Merge remote-tracking branch 'upstream/master' into the-clothing-outlet

Donglesplonge committed:
Merge remote-tracking branch 'upstream/master' into the-clothing-outlet

Donglesplonge committed:
minor map edits

adjusts the freezer room a slight bit, replaces 1 meatspike for 2 in the center of the room and puts a fridge with meat products in its place, adds a fire alarm to each area (minus the cult influenced zone).

Maurukas reviewed (changes requested):
<details>
<summary>The coldroom has atmos/firealarm issues</summary>

There is no fire alarm inside this room.
You're using `/turf/open/floor/iron/kitchen_coldroom/freezerfloor`, which starts with cold atmos. When this room is opened it instantly sets off the fire alarms. `/turf/open/floor/iron/freezer` would be a better choice here, as it's the same sprite, but without the cold atmos.

![image](https://user-images.githubusercontent.com/66576896/230245970-77552c59-a5b2-41b4-8fb8-e144c26750a6.png)
</details>

Only thing that came up. Otherwise, this looks fantastic.

Donglesplonge committed:
review requests 1

replaces the flooring in the coldroom with the proper iron/freezer tiles to prevent issues with the atmospherics, adds a fire alarm to the room just under the body storage tray.

Donglesplonge committed:
Merge branch 'master' into the-clothing-outlet
